src/interfaces/statics.rs

```diff
@@ -344,7 +349,7 @@ mod tests {
             assert!(new_partitions_mpk.contains(p));
         }
         // 3 is the size of the security level axis
-        assert_eq!(new_partitions_msk.len(), partitions_msk.len() + 3);
+        assert_eq!(new_partitions_msk.len(), partitions_msk.len() + 5);
```

Comment:
Change the comment, or put the array inside a variable and use `levels.len()`?

Reply:
thx